Lib/test/test_ssl.py

```diff
@@ -4364,14 +4364,14 @@ def test_session_handling(self):
     def test_psk(self):
         psk = bytes.fromhex('deadbeef')
 
-        client_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        client_context, server_context, _ = testing_context()
```

Comment:
Is this change needed?
Not being an OpenSSL expert, I'd rather avoid having to review it.

Reply:
It's required for this test to pass when built against AWS-LC, but I'll remove it to conform to your guidance [here](https://discuss.python.org/t/support-building-ssl-and-hashlib-modules-against-aws-lc/44505/4):

> IMO we could get to the point where you can get CPython to compile with AWS-LC without special-casing it (i.e. no Py_OPENSSL_IS_AWSLC), but relevant test cases still fail.
> You’ll still need a patch, but a more focused one.